src/prelaunch/starblaster.a

```diff
@@ -0,0 +1,32 @@
+;license:MIT
+;(c) 2019 by qkumba
+
+!cpu 6502
+!to "build/PRELAUNCH/STARBLASTER",plain
+*=$106
+
+    !source "src/prelaunch/common.a"
+
+         ;+ENABLE_ACCEL   ; can't - shows title right away
+         lda   #$60       ; hook
+         sta   $9318
+         jsr   $6BD1      ; decompress/title
+
+         ldy   #6
+-        lda   $100-1,y
+         sta   $280-1,y   ; move hook to $280
```

Comment:
If the -1 bothers you, you can change to LDY #5 and BPL, and use $100,y and $280,y. Entirely up to you.